src/keys/drivers/YubiKey.cpp

```diff
@@ -112,23 +113,36 @@ bool YubiKey::deinit()
 
 void YubiKey::detect()
 {
+    bool found = false;
+
     if (init()) {
-        for (int i = 1; i < 3; i++) {
-            YubiKey::ChallengeResult result;
-            QByteArray rand = randomGen()->randomArray(1);
-            QByteArray resp;
+        YubiKey::ChallengeResult result;
+        QByteArray rand = randomGen()->randomArray(1);
+        QByteArray resp;
 
+        // Check slot 1 and 2 for Challenge-Response HMAC capability
+        for (int i=1; i <= 2; ++ i) {
```

Comment:
@droidmonkey spaces around `=`, and no spaces after `++`

Reply:
This was clearly done by an amateur with questionable spacebar control. I'll fix it right quick :wink: